### PR TITLE
Jeff sabrina will

### PR DESCRIPTION
**Pull request by wian5430**
I want to merge my code with master

### CODE REVIEW

File: utility_functions/main.cpp [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 int main() {
     // make sure to run your functions!
+    Sum(7,10);
 }
+
+std::vector<int> v{1,2,3,4};
+
+std::vector<int> VectorPlusN(v, 5);
```
Comment (bensont) [inline utility_functions/main.cpp]: This should be included in main so the test runs. Also, std::vector<int> VectorPlusN(v, 5); is invalid syntax.
VectorPlusN() has a return that has not been instantiated. Use td::vector<int> returned_vector = VectorPlusN(v, 5);

File: utility_functions/main.cpp [hunk 2]
```diff
@@ -2,4 +2,9 @@
 
 int main() {
     // make sure to run your functions!
+    Sum(7,10);
```
Comment (bensont) [inline utility_functions/main.cpp]: This has a return that isn't accounted for. Collect the return and print the result as well as the sent numbers to Sum()

**Review by bensont — CHANGES_REQUESTED** [review]
Compilation Error:

main.cpp:10:30: error: ‘std::vector<int> VectorPlusN’ redeclared as different kind of symbol
 std::vector<int> VectorPlusN(v, 5);
                              ^
In file included from main.cpp:1:0:
ManyFunctions.h:20:18: note: previous declaration ‘std::vector<int> VectorPlusN(std::vector<int>, int)’
 std::vector<int> VectorPlusN(std::vector<int> v, int n);

The declaration in main should be changed.
Also, the main doesn't show any output to confirm that the function runs correctly.

**Review by muzny — CHANGES_REQUESTED** [review]
Please make the changes that Tyler requested by class on 2/19.

**Comment by sabrinatouch** [comment]
I want to merge my code with master

**Review by bensont — APPROVED** [review]

**Review by muzny — APPROVED** [review]
Looks good!

**Pull request merged**